test/specs/deployAndRetrieve.e2e.ts

```diff
@@ -335,49 +522,4 @@ describe('Deploy and Retrieve', async () => {
   step('Tear down and clean up the testing environment', async () => {
     await testSetup.tearDown();
   });
-
-  const runAndValidateCommand = async (
```

Comment:
Why don't we want to use the `runAndValidateCommand()` helper function anymore?

Reply:
I was wondering the same 🥹

Reply:
Sorry, merge from develop cockpit error. It's back

Thanks for catching this.